package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "create-antonella-app",
+  "version": "1.0.5",
+  "description": "",
+  "main": "src/index.js",
+  "jest": {
+    "snapshotSerializers": [
+      "enzyme-to-json/serializer"
+    ],
+    "moduleNameMapper": {
+      "\\.(css)$": "identity-obj-proxy",
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js"
+    }
+  },
+  "bin": {
+    "create-antonella-app": "./index.js"
+  },
+  "scripts": {
+    "test": "jest --verbose",
+    "test:watch": "npm run test -- --watch",
+    "start": "webpack-dev-server --hot --mode development --devtool eval-source-map",
+    "build": "webpack -p --devtool source-map"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^16.11.0",
+    "react-dom": "^16.11.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.6.4",
```

Comment:
you don't need all these dependencies for this repo - you just need jest and eslint